example/lib/screens/message_view.dart

```diff
@@ -82,28 +60,41 @@ class _MessageViewState extends State<MessageView>  {
     });
   }
 
-  onLoadStarted() {
+  onLoadEventReceived(String method, PlatformException e) {
     setState(() {
-      isLoading = true;
+      switch (method) {
+        case 'onLoadStarted':
```

Comment:
I think we need to make separate handlers for each type of event in the inbox_message_view.dart that hides the raw event data so we can do extra parsing on the data if needed. Sorta like we do for the other events.

Reply:
ok

Reply:
actually just to be sure we're on the same page, you're talking abut FlutterMethodChannel or FlutterEventChannel? Because here I switched to the method channel

Reply:
What I think we want to break up the handlers:

```
InboxMessageView(messageId: widget.messageId, 
onLoad:handleLoad, 
onError:handleError,
onClose:handleClose
etc...)
```

Make all the handlers optional.